doc/mgr/dashboard.rst

```diff
@@ -244,6 +244,19 @@ For example, a key pair can be generated with a command similar to:
    -subj "/O=IT/CN=ceph-mgr-dashboard" -days 3650 \
    -keyout dashboard.key -out dashboard.crt -extensions v3_ca
 
+.. note::
+
+   Currently, the Ceph Dashboard only supports RSA private keys for SSL/TLS
+   certificates. If you attempt to configure the dashboard with an ECDSA/EC
+   key, the module will fail to start with an error similar to:
+
+   ``MGR_MODULE_ERROR: Module 'dashboard' has failed: key type unsupported``
+
+   This limitation exists because the verification routine in the Ceph manager
```

Comment:
I believe "Manager" should start with capital case if this refers to the Manager service.